operator/pkg/sidecarinjector/sidecarinjector.go

```diff
@@ -257,6 +269,11 @@ func (m *SidecarInjector) setupMeshService() error {
 		}
 	}
 
+	m.meshService.AppContainerName = container.Name
```

Comment:
Move it to Line: 258 should be good, AppContainerName is set only when its value is empty

Reply:
Sure, updated